<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>656</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>869</width>
    <height>656</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>869</width>
    <height>656</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3DViewer 2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_name_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count_vertex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество вершин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count_polygon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество полигонов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotation">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>74</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Вращение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_movement">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Движение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertex">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Вершины</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_line">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Линии</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotation_x">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotation_y">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotation_z">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_movement_y">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_movement_z">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_movement_x">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_box_rotate_x">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>260</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="maximum">
     <double>99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_box_rotate_y">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>290</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="maximum">
     <double>99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_box_rotate_z">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>320</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="maximum">
     <double>99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_box_movement_y">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_box_movement_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_box_movement_z">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_size_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_box_type_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>Сплошная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пунктирная</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="button_select_color_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_vertex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_size_vertex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_vertex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_box_type_vertex">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>Отсутствует</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Круг</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Квадрат</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="button_select_color_point">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_projection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Проекция:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_background_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет фона:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштаб:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_box_projection_type">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>Ортогональная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Перспективная</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="button_select_background_color">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_draw">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
      color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Отрисовать</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_size_vertex">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>600</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_size_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_box_scale_model">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="WidgetDraw" name="widget_draw" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>591</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="qaction_open_file"/>
    <addaction name="qaction_clear_model"/>
    <addaction name="qaction_save_bmp"/>
    <addaction name="qaction_save_jpeg"/>
    <addaction name="qaction_save_gif"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="qaction_open_file">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="SaveSettings">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="loadSettings">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="qaction_clear_model">
   <property name="text">
    <string>Сбросить настройки</string>
   </property>
  </action>
  <action name="qaction_save_bmp">
   <property name="text">
    <string>Сохранить .bmp</string>
   </property>
  </action>
  <action name="qaction_save_jpeg">
   <property name="text">
    <string>Сохранить .jpeg</string>
   </property>
  </action>
  <action name="qaction_save_gif">
   <property name="text">
    <string>Сохранить .gif</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetDraw</class>
   <extends>QWidget</extends>
   <header>../../../src/front/widget_draw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
